docs-app/tests/helpers/index.js

```diff
@@ -0,0 +1,42 @@
+import {
+  setupApplicationTest as upstreamSetupApplicationTest,
+  setupRenderingTest as upstreamSetupRenderingTest,
+  setupTest as upstreamSetupTest,
+} from 'ember-qunit';
+
+// This file exists to provide wrappers around ember-qunit's / ember-mocha's
+// test setup functions. This way, you can easily extend the setup that is
+// needed per test type.
+
+function setupApplicationTest(hooks, options) {
+  upstreamSetupApplicationTest(hooks, options);
+
+  // Additional setup for application tests can be done here.
+  //
+  // For example, if you need an authenticated session for each
+  // application test, you could do:
+  //
+  // hooks.beforeEach(async function () {
+  //   await authenticateSession(); // ember-simple-auth
+  // });
+  //
+  // This is also a good place to call test setup functions coming
+  // from other addons:
+  //
+  // setupIntl(hooks); // ember-intl
+  // setupMirage(hooks); // ember-cli-mirage
+}
+
+function setupRenderingTest(hooks, options) {
+  upstreamSetupRenderingTest(hooks, options);
+
+  // Additional setup for rendering tests can be done here.
+}
+
+function setupTest(hooks, options) {
+  upstreamSetupTest(hooks, options);
+
+  // Additional setup for unit tests can be done here.
+}
+
+export { setupApplicationTest, setupRenderingTest, setupTest };
```

Comment:
Totally non-blocking quesiton: We have all of the same in `toucan-core` under `docs-app/tests`, but I'm curious if it makes sense for our docs-apps to even have the `tests/` directory?  We normally use the `test-app` for testing, so just curious on thoughts around this.

Reply:
over time docs may have demos complex enough where you want them tested

Reply:
Ah, that makes sense 👍

Reply:
Yeah, and I would also like to add even just a very simple smoke test for the docs app, mainly checking that the build does not blow up, and at least something is rendered. This has proven to be useful in other projects like ember-bootstrap, to make sure that e.g. automated renovate updates don't accidentally the docs app!